### DISCUSSION TITLE
Upgrade to ImageSharp 2.0 to reintroduce WebP support

### DISCUSSION THREAD

**Discussion by maartenmensink** [umbraco/Umbraco-CMS#11604] [Features and ideas]
With the upgrade to Umbraco 9 we lost support for webp due to the fact that ImageSharp has no support for WebP in version `1.x` 

The milestone for version `2.0` is almost complete and can be viewed at. https://github.com/SixLabors/ImageSharp/issues?q=is%3Aopen+is%3Aissue+milestone%3A2.0.0

cc @nul800sebastiaan

**Comment by bielu** [comment]
@maartenmensink not sure if v2 follow same licence pattern for OSS, so it should be check before update umbraco to support imagesharp to v2 :)

**Reply by Nuklon** [reply]
Still Apache 2.0.

**Comment by maartenmensink** [comment]
Seems initial step are taking in https://github.com/umbraco/Umbraco-CMS/pull/11991

**Comment by ronaldbarendse** [comment]
Umbraco 10 will ship with updated ImageSharp/ImageSharp.Web v2 dependencies, as we can't upgrade without breaking changes, see PR https://github.com/umbraco/Umbraco-CMS/pull/12185.
